### PR TITLE
[HOPSWORKS-1702] register user as admin

### PR DESCRIPTION
**Pull request by ErmiasG**

### CODE REVIEW

File: docs/admin_guide/user-administration.rst [hunk 1]
```diff
@@ -127,4 +127,74 @@ The new autogenerated password will be shown. Make sure to copy it before closin
...
+.. figure:: ../imgs/admin/register-remote-user.png
+  :alt: Register remote user
+  :target: `register-remote-user.png`_
+  :align: center
+  :figclass: align-center
+ 
+  Register remote user
+ 
+To register a remote user you need a 'UUID' that is the universally unique id of the remote user in LDAP or Kerberos.
+To find this id in Active directory go to users and computers management window and right click on the user you want to
```
Comment (tkakantousis) [inline docs/admin_guide/user-administration.rst:179]:
```suggestion
To find this id in Active directory go to "users and computers" management window and right click on the user you want to
```

File: docs/admin_guide/user-administration.rst [hunk 2]
```diff
@@ -127,4 +127,74 @@ The new autogenerated password will be shown. Make sure to copy it before closin
...
+.. _ad-objectuuid.png: ../_images/admin/ad-objectuuid.png
+.. figure:: ../imgs/admin/ad-objectuuid.png
+  :alt: Active directory objectUUID
+  :target: `ad-objectuuid.png`_
+  :align: center
+  :figclass: align-center
+ 
+  Active directory objectUUID
+ 
+To find the uuid in LDAP run 'ldapsearch' as shown in the image below.
```
Comment (tkakantousis) [inline docs/admin_guide/user-administration.rst:191]:
```suggestion
To find the `UUID` in LDAP run 'ldapsearch' as shown in the image below.
```

File: docs/admin_guide/user-administration.rst [hunk 3]
```diff
@@ -127,4 +127,74 @@ The new autogenerated password will be shown. Make sure to copy it before closin
...
+.. _register-remote-user.png: ../_images/admin/register-remote-user.png
+.. figure:: ../imgs/admin/register-remote-user.png
+  :alt: Register remote user
+  :target: `register-remote-user.png`_
+  :align: center
+  :figclass: align-center
+ 
+  Register remote user
+ 
+To register a remote user you need a 'UUID' that is the universally unique id of the remote user in LDAP or Kerberos.
```
Comment (tkakantousis) [inline docs/admin_guide/user-administration.rst:178]:
```suggestion
To register a remote user you need a `UUID` that is the universally unique id of the remote user in LDAP or Kerberos.
```

**Review by tkakantousis — CHANGES_REQUESTED** [review]

**Comment by ErmiasG** [comment]
logicalclocks/hops-testing-ee/pull/158

**Review by tkakantousis — APPROVED** [review]

**Pull request merged**